signingscript/requirements/base.in

```diff
@@ -6,6 +6,7 @@ mohawk>=1.0.0
 winsign==2.2.4
 macholib>=1.11
 jsonschema>=4.0.0
-# for mozbuild
+
+# for mozbuild - remove after the following bug is fixed: https://bugzilla.mozilla.org/show_bug.cgi?id=1831648
```

Comment:
I don't think this comment is accurate.  We vendor mozbuild so we'll still need to list its dependencies here.

Reply:
🤔 Shouldn't it `pip install -e ..../mozbuild` ? 

Had a better look at it, and it looks like there's some confusion and copy/pasta going on.

It's included in iscript, but I think it's leftover copy/paste, as it's not used there at all.

It's vendored in the [root folder](https://github.com/mozilla-releng/scriptworker-scripts/tree/master/vendored), **AND** in [signingscript](https://github.com/mozilla-releng/scriptworker-scripts/tree/master/signingscript/src/signingscript/vendored), and looks like it [uses the one in signingscript/vendored](https://github.com/mozilla-releng/scriptworker-scripts/blob/master/signingscript/src/signingscript/sign.py#L46) for a single function (`safe_extract`)

Thoughts on how to better manage this?
As a first step I think we should remove the vendored folder from root and iscript req.

Reply:
If we're literally just using it for one function let's just copy/paste that function (and whatever it depends on) somewhere in signingscript and avoid this whole mess...

Reply:
signingscript and iscript use mozjar in addition to safe_extract.